BlueMapCore/src/main/java/de/bluecolored/bluemap/core/render/lowres/LowresModelManager.java

```diff
@@ -189,10 +187,20 @@ private LowresModel getModel(UUID world, Vector2i tile) {
 						try (FileInputStream fis = new FileInputStream(modelFile)) {
 							InputStream is = fis;
 							if (useGzip) is = new GZIPInputStream(is);
+							byte[] buffer = new byte[1024];
 							
-							String json = IOUtils.toString(is, StandardCharsets.UTF_8);	
+							ByteArrayOutputStream out = new ByteArrayOutputStream();
+
+							int len;
+							while ((len = is.read(buffer)) > 0) {
+								out.write(buffer, 0, len);
+							}
+
+							is.close();
+							out.close();
+							byte[] data = out.toByteArray();
```

Comment:
Since we are depending on apache.commons we could just use `byte[] data = IOUtils.toByteArray(is);` here